projects/ngx-wig/src/lib/config.ts

```diff
@@ -7,7 +7,7 @@ export type TButton = {
   command?: string;
   styleClass?: string;
   pluginName?: string;
-  isComplex?: boolean;
+  isPlugin?: boolean;
```

Comment:
Would it worth to get rid of the `boolean` property and check the existence of a plugin with `pluginName` property?

Reply:
true, I even don't know whether we need second one.
I'll cleanup both